src/main/java/it/pagopa/transactions/utils/EcsEncoderLogMasker.java

```diff
@@ -15,6 +19,12 @@ public class EcsEncoderLogMasker extends EcsEncoder {
     private Pattern patternString;
     private final List<String> maskPatterns = new ArrayList<>();
 
+    private ObjectMapper objectMapper;
+
+    public EcsEncoderLogMasker() {
```

Comment:
Add package private constructor with ObjectMapper as input parameter so that you don't have to use reflections into tests